app/views/document/AddAdditionalInformationYesNoView.scala.html

```diff
@@ -45,7 +45,8 @@
                 heading = messages("document.addAdditionalInformationYesNo.heading"),
                 caption = Some(messages("document.caption")),
                 additionalHtml = messages("document.addAdditionalInformationYesNo.paragraph").toParagraph
-            )
+            ),
+            hint = Some(HtmlContent(messages("document.addAdditionalInformationYesNo.hint")))
```

Comment:
`HtmlContent` is designed for raw html. Use `Text` as the content type when the input is a string